aikau/src/main/resources/alfresco/menus/_AlfMenuItemMixin.js

```diff
@@ -164,7 +164,7 @@ define(["dojo/_base/declare",
          }
          if (!this.iconAltText && this.title)
          {
-            this.iconAltText = originalLabel;
+            this.iconAltText = this.message(originalLabel);
```

Comment:
Looking at the original code, I'm wondering if my intention (and the more appropriate behaviour) would be to use `this.title` for the `iconAltText`... because it's likely that the title would be more informative than the label. I'm happy to merge this (as it's a fairly minor change I'm not worried about a unit test), but just wanted to flag up that it might be better to use the title rather than the label).